models/articles.model.js

```diff
@@ -19,3 +19,15 @@ exports.selectArticleById = ( id ) => {
     })
 }
 
+exports.selectUpdatedArticle = (article_id, newVote) => {
+  const queryString = `UPDATE articles SET votes = votes + ${newVote} WHERE article_id = $1 RETURNING *`;
```

Comment:
We have opened ourselves up to SQL injection here, always avoid template literals if you can. Stick to the dollar syntax.

Reply:
Cool. Thanks :)